internal/packer_registry/registry.go

```diff
@@ -0,0 +1,140 @@
+package packer_registry
+
+import (
+	"context"
+	"errors"
+
+	"github.com/go-openapi/runtime"
+	packerSvc "github.com/hashicorp/hcp-sdk-go/clients/cloud-packer-service/preview/2021-04-30/client/packer_service"
+	"github.com/hashicorp/hcp-sdk-go/clients/cloud-packer-service/preview/2021-04-30/models"
+	"google.golang.org/grpc/codes"
+)
+
+// CreateBucket creates a bucket on a HCP Packer Registry.
+func CreateBucket(ctx context.Context, client *Client, input *models.HashicorpCloudPackerCreateBucketRequest) (string, error) {
```

Comment:
Is there a reason you didn't pass the `Client` as a receiver here? I think it makes sense as a wrapper of `client.Packer.CreateBucket(params, nil, func(*runtime.ClientOperation) {})`